packages/angular-workspace/nimble-angular/styles/fonts.scss

```
@@ -1,4 +1,4 @@
 // Nimble Angular Fonts SCSS
-// Only @import once in the global styles.scss
+// Only @forward/@use once in the global styles.scss
```

fonts don't define variables so I don't think there is a situation where an application forwards the fonts. Did you find something?

Proposing we align with the docs:
```suggestion
// Only @use once in the global styles.scss
```